ui/src/dashboards/components/dashboard_index/Table.tsx

```diff
@@ -39,79 +37,64 @@ interface DispatchProps {
   getLabels: typeof getLabels
 }
 
-type SortKey = keyof Dashboard | 'meta.updatedAt'
-
 type Props = OwnProps & StateProps & DispatchProps & WithRouterProps
 
-class DashboardsTable extends PureComponent<Props, State> {
-  state: State = {
-    sortKey: 'name',
-    sortDirection: Sort.Ascending,
-    sortType: SortTypes.String,
-  }
-
+class DashboardsTable extends PureComponent<Props> {
   public componentDidMount() {
     this.props.getDashboards()
     this.props.getLabels()
   }
 
   public render() {
-    const {status, dashboards, filterComponent, onFilterChange} = this.props
-
-    const {sortKey, sortDirection, sortType} = this.state
-
-    let body
+    const {
+      status,
+      dashboards,
+      onFilterChange,
+      sortKey,
+      sortDirection,
+      sortType,
+      onCreateDashboard,
+      searchTerm,
+    } = this.props
 
     if (status === RemoteDataState.Done && !dashboards.length) {
-      body = (
-        <ResourceList.Body emptyState={null}>
-          {this.emptyState}
-        </ResourceList.Body>
-      )
-    } else {
-      body = (
-        <ResourceList.Body style={{height: '100%'}} emptyState={null}>
-          <DashboardCards
-            dashboards={dashboards}
-            sortKey={sortKey}
-            sortDirection={sortDirection}
-            sortType={sortType}
-            onFilterChange={onFilterChange}
+      if (searchTerm) {
```

Comment:
This is getting gnarly enough were it might be better as its own component.  `DashboardEmptyState`